src/cmd/manage_nsfs.js

```diff
@@ -352,17 +352,11 @@ async function fetch_existing_account_data(action, target, decrypt_secret_key) {
     const options = { show_secrets: true, decrypt_secret_key };
     let source;
     try {
-        source = target.name ?
-            await config_fs.get_account_by_name(target.name, options) :
-            await config_fs.get_account_by_access_key(target.access_keys[0].access_key, options);
+        source = await config_fs.get_account_by_name(target.name, options);
     } catch (err) {
-        dbg.log1('NSFS Manage command: Could not find account', target, err);
+        dbg.warn('fetch_existing_account_data: got an error on', target.name, err);
```

Comment:
maybe more readable message,
`fetch_existing_account_data: account with name %s could not find`

Reply:
I will change the string to a template, hoping it will be a more readable message, but I can't add the "could not find" because it is an error that we receive after "get_account_by_name" and "ENOENT" is only one option, can be more causes.

Reply:
templet looks good to me